src/main/java/org/takes/facets/auth/social/PsGithub.java

```diff
@@ -122,9 +143,12 @@ private static Identity fetch(final String token) throws IOException {
      */
     private String token(final String home, final String code)
         throws IOException {
-        // @checkstyle LineLength (1 line)
-        final String uri = new Href("https://github.com/login/oauth/access_token")
-            .with("client_id", this.app)
+        final String uri = new Href(
```

Comment:
@dmzaytsev Same here, I don't like this design. Why not do it as originally suggested in #42?